cmd/git/main.go

```diff
@@ -164,6 +164,13 @@ func runGitClone(ctx context.Context) error {
 		return &ExitError{Code: 101, Message: "the 'target' argument must not be empty"}
 	}
 
+	// sanity-check the endpoint, if hostname extraction fails, ignore that failure
```

Comment:
Nit: Trying to avoid ableist language.
```suggestion
	// check the endpoint, if hostname extraction fails, ignore that failure
```